pom.xml

```diff
@@ -152,6 +148,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Why pinning calcite?

Reply:
We should move this to dependency management section if a different version is being brought in.

Reply:
It's not a part of the parent or any other dependency, it's the only place where it is coming from.

Reply:
Calcite should come from hive exec as I recall. Otherwise why do we need to pin the version here?